brightchain-lib/src/lib/blockMetadata.ts

```
@@ -65,26 +68,26 @@ export class BlockMetadata implements IBlockMetadata {
    * @returns Parsed metadata object
    * @throws If JSON is invalid or required fields are missing
    */
-  public static fromJson(json: string): BlockMetadata {
+  public static fromJson<
+    B extends BlockMetadata,
+    T extends Record<string, any>,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)